### ISSUE TITLE
Stopped working on Invidious

### ISSUE THREAD

**Issue by acidghost** [ajayyy/SponsorBlock#1706]
It's not working on any of the default invidious instances. It says:
> No YouTube video found. If this is incorrect, refresh the tab.

Of course refreshing didn't work.

I also tried disabling and re-enabling 3rd party YT sites and manually adding an existing (default) instance as well as other instances.

Host: macOS 13.2.1 arm64
Browser: Firefox 111.0.1
SponsorBlock: v5.3.0

Let me know if I can provide more info.

**Comment by ajayyy** [comment]
Yes sorry Mozilla is taking their time approving my bug fix for this. You can either wait or downgrade

**Comment by tuxayo** [comment]
Anyone else still having the same issue since a few weeks? (date blurry in my mind)
Version 5.4.8
Firefox dev edition: 115.0b6 (also happened on previous version, which is now the stable Firefox version)
